packages/gluestick-plugin-bunyan/src/server.js

```diff
@@ -19,19 +20,29 @@ export type Logger = {
   error: Function,
 };
 
+type PluginUtilities = {
+  requireModule: Function,
+};
+
 const defaultSettings: Options = {
   name: 'default name',
 };
 
-const bunyanPlugin = (): { logger: Logger } => {
-  let options: Options = defaultSettings;
+type BunyanPlugin = (
+  opts: Object,
+  pluginUtilities: PluginUtilities,
+) => { logger?: Logger };
+
+const bunyanPlugin: BunyanPlugin = (opts, { requireModule }) => {
```

Comment:
I wonder if the future of plugins should include some first-class configuration stuff.